src/declaration.d

```diff
@@ -507,6 +507,27 @@ public:
         return false;
     }
 
+version(IN_LLVM)
+{
+    override void semantic3(Scope *sc)
```

Comment:
@klickverbot Did this end up having something to do with the llvm linker bug?  Or is it still needed?